### DISCUSSION TITLE
how to execute commands  using on_equip?

### DISCUSSION THREAD

**Discussion by Grxxney** [bridge-core/bridge.#338] [Add-On Q&A] [answered]
I want to execute an command (for example: /enchant @s infinity 1) when an player equip on (component: bridge:item_equipped_sensor -> on_quip) my item but I don't know if that's possible. 

If yes than how I can do that?

**Comment by Chikorita-Lover** [comment, marked as answer]
You'd probably be better off using a behavior animation in the player behavior file. You could run this animation with the MoLang query `query.get_equipped_item_name == 'banana_split'` (if your item is something like `bridge:banana_split`), and have the animation run the `/enchant` command.
